.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2023 Shukant Pal
+# SPDX-License-Identifier: Apache-2.0
+name: Publish to PyPI
+
+on:
+  workflow_dispatch:
```

Comment:
Please add

```
push:
    # Also trigger on push of release tags to any branch. Useful
    # for testing release builds before merging to main.
    tags:
      - 'v[0-9]+\.[0-9]+\.[0-9]+'
      - 'v[0-9]+\.[0-9]+\.[0-9]+-*'
```

The last line is for alphas, betas and release candidates. Do you think we should avoid deploying pre-releases?

Reply:
Done

Reply:
I think it’s ok to deploy pre-releases. You will have to update the version in [this file](https://github.com/ShukantPal/KTX-Software/blob/5505ac4e902d5fdf7675c82457fa37217bcc7c46/interface/python_binding/setup.py#L7).

Reply:
Can `setup.py` get the version from the libktx version file, `lib/version.h`? So far I've managed to avoid having to manually update any versions prior to making releases. Everything comes automatically from the git tag name. I even managed to coerce Maven's `pom.xml` into doing this for the JNI build.

Reply:
Did you do that in the CMake build?

Reply:
Thanks for the pointer. Since `setup.py` is a Python script, it can read environment variables. I injected `LIBKTX_VERSION` using the existing CMake build.